src/src/main/java/com/iwaconsolti/calculator02/AppRunner.java

```diff
@@ -0,0 +1,54 @@
+package com.mx.iwaconsolti.calculator02;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+import java.util.Scanner;
+
+@Component
+public class AppRunner implements CommandLineRunner {
+    private final AdditionService additionService;
```

Comment:
You should make all fo these IOperations type so in the constructor you are required to use the Qualifer name to identify